1.9/storage/external-storage.md

```diff
@@ -86,7 +86,7 @@ You can specify an external volume in your Marathon app definition. [Learn more
       "instances": 1,
       "cpus": 0.1,
       "mem": 32,
-      "cmd": "/usr/bin/tail -f /dev/null",
+      "cmd": "mkdir my-test-vol && touch my-test-vol/test.txt && ls my-test-vol/",
```

Comment:
I believe this should be 
```
mkdir test-rexray-volume/test-dir && touch test-rexray-volume/test-dir/test.txt && ls test-rexray-volume/test-dir/
```

The `my-test-vol` property that is specified for this volume is the name of the volume in eg., your AWS account, and is unrelated to the directory path in the container. The directory path in the container is given by the `containerPath` property, which in this case is `test-rexray-volume`.